internal/runnerinstall/ecs.go

```diff
@@ -411,19 +420,16 @@ DeleteFileSystem:
 	describeMountTargetsResp, err := efsSvc.DescribeMountTargets(&efs.DescribeMountTargetsInput{
 		FileSystemId: fileSystemId,
 	})
-	if err != nil {
-		return err
-	}
```

Comment:
Do we not need to check this error?

Reply:
We do! I lost this while moving other things around, but fixed up!